app/src/main/java/edu/calvin/cs262/project01/Login.java

```diff
@@ -97,4 +106,16 @@ public void handleClickProfile(MenuItem item) {
         Intent profile = new Intent(this, Profile.class);
         this.startActivity(profile);
     }
+
+
+
+
+    /**
+     * handleAboutUs should start aboutUs activity and show our vision statement
+     */
+
+    public void openForgotPasswordPage() {
+        Intent forgotPassword= new Intent(this, ForgotPassword_Page.class);
```

Comment:
fix documentation
handleAbout us comment is used above, and does not match openForgotPasswordPage